src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,226 @@
+
+## Motion recording data
+
+{{ MACROS\_\__make_filename_template(
+"raw",
+datatypes=\["motion"],
+suffixes=\["motion", "channels", "events"])
+}}
+
+Template:
+
+```markdown
+└─ sub-<label>\
+  └─ \[ses-<label>]\
+    └─ motion\
+	  ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_motion.tsv
+      ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_motion.json
+      ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_channels.tsv
+      ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.tsv
+      └─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.json
+```
+
+A wide variety of motion capture systems are used in human research, resulting in different native data formats. This BIDS extension proposal (BEP) specifically deals any standard outputs from motion capture systems such as positions, orientations or acceleration among others. For camera-based motion capture the raw camera footage falls out of scope. To share motion data recorded using an optical system, typically the position or orientation or likewise system output is formatted in BIDS, not the raw camera footage.
+
+The extension is not limited to motion data in physical space but also encompasses simulated movement in virtual space, as far as these are comparable to movements in physical space. The extension is also not limited to the positions/orientations of human body parts, other dynamic objects (physical or virtual) in the environment
+whose motion is tracked may be included as additional indications of motion.
+
+For the current BEP, motion data coming from one tracking system MUST be stored in a `*_motion.tsv` file. A tracking system is defined as a group of tracking devices that share hardware properties (same recording device) and/or software properties (for example same sampling rates). For example if the position of multiple optical markers is processed via one recording unit, this would be defined as a tracking system. Note that it is not uncommon to have multiple tracking systems to record at the same time. Each tracking system should have its own `*tracksys-<label>_motion.tsv` file. One column in the `*tracksys-<label>_motion.tsv` file is intended to represent one data channel. The ordering of columns has to match the order of rows in the `*channels.tsv` file to allow a unambiguous assignment. All relevant metadata about a tracking systems is stored in accompanying sidecar `*tracksys-<label>_motion.json` file. The source data from each tracking system in their original format, if different from `.tsv`, can be stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data). The original data format might additionally hold more metadata than currently being specified in the `*_motion.json` file.
```

Comment:
consistency: you sometimes write `*_foo.tsv` and at other times  `*foo.tsv` -- take a look at how it is generally done in the spec and adopt that policy for here as well.

(and if it's inconsistent in the spec, please raise an issue :see_no_evil: )

---

you introduce `tracksys-<label>` quite abruptly, I think a short sentence would be nice to explain that the use of `<label>` allows to distinguish the multiple tracking systems.